packages/quantic/force-app/main/default/lwc/quanticFoldedResultList/quanticFoldedResultList.js

```diff
@@ -99,7 +99,7 @@ export default class QuanticFoldedResultList extends LightningElement {
       options: {
         fieldsToInclude: this.fields,
         folding: {
-          numberOfFoldedResults: this.numberOfFoldedResults,
+          numberOfFoldedResults: Number(this.numberOfFoldedResults),
```

Comment:
What about using something like this?

```
_numberOfFoldedResults = 2;
@api
get numberOfFoldedResults() {
    return this._numberOfFoldedResults;
}
set numberOfFoldedResults(value) {
    const num = parseInt(value, 10);
    this._numberOfFoldedResults = (num !== NaN) ? num : 2;
}
```

That way you ensure it's a number, otherwise you go back to the default?

Reply:
Yeah this could work perfectly as well 👌
My only small concern is about consistency, because  we currently do not have this strategy of falling back to the default value  when an invalid property is passed to a Quantic component. 

IMO the end user should be very aware that the value  introduced to a property is invalid. Falling back to the default value   can make invalid properties, easily, go under the radar/unnoticed which can be a bit problematic.

Going with the implementation in this PR was my first choice mainly because  we currently using the same logic with other numeric public properties like, for example, in the Quantic Facet component here: https://github.com/coveo/ui-kit/blob/71c783e9d989573f40959ff0aeb93b5c5afdab5c/packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js#L239

Reply:
this is a good point: 

> IMO the end user should be very aware that the value introduced to a property is invalid. Falling back to the default value can make invalid properties, easily, go under the radar/unnoticed which can be a bit problematic.

Reply:
Ok well the fallback was just an idea, but do you prefer using the `parseInt(value, 10)` to the `Number(value)` method?

I would prefer it I think because using the Number constructor could give you non Integer AFAIK, so you'd have 3.5 numberOfFoldedResults potentially?